src/VaultV2.sol

```diff
@@ -40,8 +40,8 @@ import {ISharesGate, IReceiveAssetsGate, ISendAssetsGate} from "./interfaces/IGa
 /// @dev The Vic must not call totalAssets() because it will try to accrue interest, but instead use the argument
 /// _totalAssets that is passed.
 ///
-/// FIRST TOTAL ASSETS
-/// @dev The variable firstTotalAssets tracks the total assets after the first interest accrual of the transaction.
+/// UNDERESTIMATED TOTAL ASSETS
+/// @dev The variable underestimatedTotalAssets is an underestimation of the current value of `_totalAssets`..
```

Comment:
should we explain how it behaves?

Reply:
I tried something, let me know what you think